grind/python/disttest/isolate.py

```diff
@@ -97,14 +98,18 @@ def _generate_run_script_contents(self):
         exit $rc
     fi
 }
-export JAVA7_BUILD=true
-. /opt/toolchain/toolchain.sh
+
 run ./unpack.sh # Generated by packager
 source environment.source # Init runtime environment
+"""
+        contents += """
+jdks=(/usr/lib/jvm/java-{java_version}-*)
+export JAVA_HOME=${{jdks[0]}}
```

Comment:
where we assign JAVA_HOME, e.g. should it be:

export JAVA_HOME="${{jdks[0]}}"

I don't think it matters that much since these paths won't have a space, but I think good bash style?